OpenOversight/app/utils.py

```diff
@@ -460,20 +478,21 @@ def rm_dirs():
         return hash_found
 
     # Generate new filename
-    file_extension = original_filename.split('.')[-1]
+    file_extension = filename.split('.')[-1]
     new_filename = '{}.{}'.format(hash_img, file_extension)
 
     # Upload file from local filesystem to S3 bucket and delete locally
     try:
-        url = upload_file(safe_local_path, original_filename,
+        url = upload_file(safe_local_path, filename,
                           new_filename)
         rm_dirs()
         # Update the database to add the image
+        # If we're cropping an image, get data from the original image,
+        # else, use data given to us.
         new_image = Image(filepath=url, hash_img=hash_img, is_tagged=True,
```

Comment:
There's a bug here that when I'm uploading photos from the main Submit Images page (not from officer profile), this causes the images to be marked as tagged when that isn't the case. They subsequently don't show up in the tagging queue.